diff --git a/frontend/libs/console/legacy-ce/src/lib/components/Main/Main.js b/frontend/libs/console/legacy-ce/src/lib/components/Main/Main.js
index bacc6fb3ad..2c6983ec62 100644
--- a/frontend/libs/console/legacy-ce/src/lib/components/Main/Main.js
+++ b/frontend/libs/console/legacy-ce/src/lib/components/Main/Main.js
@@ -438,45 +438,8 @@ class Main extends React.Component {
                   </Link>
                 </div>
                 {/* Compensate legacy styles directly with style attribute */}
-                <div className={styles.proWrapper} style={{ padding: '0' }}>
-                  <div className={itemContainerStyle}>
-                    <div
-                      className={clsx(
-                        linkStyle,
-                        isPopUpOpen && activeLinkStyle
-                      )}
-                      onClick={this.onProIconClick}
-                    >
-                      <span className="text-sm self-baseline">
-                        <FaInfoCircle />
-                      </span>
-                      <span className="uppercase text-left">CLOUD</span>
-                    </div>
-                  </div>
-                  {isPopUpOpen && <ProPopup toggleOpen={this.toggleProPopup} />}
-                </div>
 
                 <Help isSelected={currentActiveBlock === 'support'} />
-                <NotificationSection
-                  isDropDownOpen={this.state.isDropdownOpen}
-                  closeDropDown={this.closeDropDown}
-                  toggleDropDown={this.toggleDropDown}
-                />
-                {!this.state.loveConsentState.isDismissed ? (
-                  <div className="bootstrap-jail">
-                    <div
-                      id="dropdown_wrapper"
-                      className={`self-stretch h-full ${
-                        this.state.isLoveSectionOpen ? 'open' : ''
-                      }`}
-                    >
-                      <LoveSection
-                        closeLoveSection={this.closeLoveSection}
-                        toggleLoveSection={this.toggleLoveSection}
-                      />
-                    </div>
-                  </div>
-                ) : null}
               </div>
             </div>
           </div>
